src/main.rs

```diff
@@ -2,11 +2,12 @@ use clap::StructOpt;
 use std::io::stdin;
 use toipe::config::ToipeConfig;
 use toipe::Toipe;
+use toipe::ToipeError;
 
-fn main() {
+fn main() -> Result<(), ToipeError> {
     let config = ToipeConfig::parse();
 
-    let mut toipe = Toipe::new(config).unwrap();
+    let mut toipe = Toipe::new(config)?;
 
     let stdin = stdin();
 
```

Comment:
The `unwrap()` below on line 17 can also be replaced with a `?` with this change

Reply:
okay cool. done